content/applications/hr/fleet/new_vehicle.rst

```diff
@@ -36,84 +49,248 @@ This section of the vehicle form relates to the person who is currently driving
+  options are :guilabel:`Doctor`, :guilabel:`Madam`, :guilabel:`Miss`, :guilabel:`Mister`, and
+  :guilabel:`Professor`.
+
+  If :guilabel:`Company` is selected for the :guilabel:`Individual` or :guilabel:`Company` field,
+  this field does not appear.
+- :guilabel:`Tags`: using the drop-down menu, select any tags that apply to the driver or company.
+
+  To add a new tag, type in the tag, then click :guilabel:`Create "tag"`.
+
+  There is no limit to the number of tags that can be selected.
+
+.. image:: new_vehicle/create-driver.png
+   :align: center
+   :alt: The top portion of the create driver form.
+
+Contacts & addresses tab
+************************
+
+After completing the top-half of the :guilabel:`Create Driver` or :guilabel:`Create Future Driver`
+form, add any other contacts and addresses associated with the driver or company in this tab.
+
+To add a new contact, click the :guilabel:`Add` button, and a :guilabel:`Create Contact` pop-up
+window appears.
+
+Enter the following information on the form:
+
+- :guilabel:`Contact Type`: select the type of contact being added. Click the radio button next to
+  the desired type. The default options are:
+
+  - :guilabel:`Contact`: select this option to add general contact details for employees of the
+    associated company.
+  - :guilabel:`Invoice Address`: select this option to add a preferred address for all invoices.
+    When added to the form, this address is selected by default when sending an invoice to the
+    associated company.
+  - :guilabel:`Delivery Address`: select this option to add a preferred address for all deliveries.
+    When added to the form, this address is selected by default when delivering an order to the
+    associated company.
+  - :guilabel:`Follow-up Address`: select this option to add a preferred address for all follow-up
+    correspondence. When added to the form, this address is selected by default when sending
+    reminders about overdue invoices.
+  - :guilabel:`Other Address`: select this option to add any other necessary addresses for the
+    company.
+
+.. image:: new_vehicle/create-contact.png
+   :align: center
+   :alt: The create contact form with all parts filled in.
+
+Depending on the :guilabel:`Contact Type`, some optional fields may not be visible. The available
+fields are identical to the fields in the :ref:`general information
+<fleet/new_vehicle/general-info>` section of the new driver form.
+
+Add any notes to the :guilabel:`Internal notes...` section of the form.
+
+After entering all of the information, click either :guilabel:`Save & Close` to add the one new
+contact, or :guilabel:`Save & New` to add the current address record and create another address
+record.
+
+As contacts are added to this tab, each contact appears in a separate box, with an icon indicating
+what type of contact is listed.
+
+.. example::
+   An :guilabel:`Invoice Address` displays a dollar sign icon inside that specific address box,
```

Comment:
is there any dollar sign emoji that could possibly be used here?

Reply:
Sure! It didn't occur to me since I thought of that as a universally understood icon, but you are right- a nice add.